src/triage/component/catwalk/predictors.py

```diff
@@ -288,7 +288,9 @@ def predict(self, model_id, matrix_store, misc_db_parameters, train_matrix_colum
             elif self.rank_order == 'worst':
                 df.sort_values(by=["score", "label_value"], inplace=True, ascending=[False,True], na_position='first')
             elif self.rank_order == 'random':
-                df.sample(frac=1).sort_values(by='score', inplace=True, ascending=False)
+                df['random'] = np.random.rand(len(df))
+                df.sort_values(by=['score', 'random'], inplace=True, ascending=[False, False])
+                df.drop('random', axis=1)
```

Comment:
Seems fine to me (so long as `df` is guaranteed to not have an existing `random` column, which I believe should be true here...). Does this end up being more efficient that doing it in one line with sample and re-assigning (certainly lower memory usage, but this dataframe shouldn't be particularly huge)?